### ISSUE TITLE
Clear Console not working on Windows

### ISSUE THREAD

**Issue by Alex-np** [spyder-ide/spyder#10762]
### What steps will reproduce the problem?

%clear
The system cannot find the path specified.

## Versions

* Spyder version: 4.0.0b5 
* Python version: 3.7.3
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```
cloudpickle >=0.5.0          :  1.2.1 (OK)
pygments >=2.0               :  2.4.2 (OK)
qtconsole >=4.5.5            :  4.5.5 (OK)
nbconvert >=4.0              :  5.5.0 (OK)
sphinx >=0.6.6               :  2.1.2 (OK)
pylint >=0.25                :  2.3.1 (OK)
psutil >=0.3                 :  5.6.3 (OK)
qtawesome >=0.5.7            :  0.5.7 (OK)
qtpy >=1.5.0                 :  1.8.0 (OK)
pickleshare >=0.4            :  0.7.5 (OK)
zmq >=17                     :  18.0.0 (OK)
chardet >=2.0.0              :  3.0.4 (OK)
numpydoc >=0.6.0             :  0.9.1 (OK)
spyder_kernels >=1.5.0;<2.0.0:  1.5.0 (OK)
qdarkstyle >=2.7             :  2.7 (OK)
atomicwrites >=1.2.0         :  1.3.0 (OK)
diff_match_patch >=20181111  :  20181111 (OK)
watchdog                     :  None (OK)
keyring                      :  None (OK)
pexpect >=4.4.0              :  4.7.0 (OK)
pympler                      :  None (OK)
sympy >=0.7.3                :  1.4 (OK)
cython >=0.21                :  0.29.12 (OK)
IPython >=4.0                :  7.6.1 (OK)
matplotlib >=2.0.0           :  3.1.0 (OK)
pandas >=0.13.1              :  0.24.2 (OK)
numpy >=1.7                  :  1.16.4 (OK)
scipy >=0.17.0               :  1.2.1 (OK)
pyls >=0.28.2;<0.29.0        :  0.28.3 (OK)
rtree >=0.8.3                :  0.8.3 (OK)
```

**Comment by ccordoba12** [comment]
This happens when your current working directory is a network mount. So avoid that until we fix it.

**Comment by darth3PO** [comment]
Same issue here with Spyder 3.3.6 as well. All other package versions being the same. 

Things do work properly if the drive is mounted `X:\Folder\To\Project` but do not with just a network folder `\\server.com\with\additional\Folder\To\Project\ `

**Comment by ccordoba12** [comment]
This looks like a bug in IPython. We'll submit a pull request there to fix it.

**Comment by dalthviz** [comment]
Checking the issues of IPython seems like this is related to ipython/ipython#11488

**Comment by ccordoba12** [comment]
Great work finding the issue there @dalthviz! Then, as I suspected, we need to submit a PR there to fix things here. Please do that.

**Comment by dalthviz** [comment]
Doing some testing, and after reading more carefully the issue in IPython (where the user specifies he is using Spyder 3.3.1), seems like the issue is actually in QtConsole and it's not only affecting the `clear` magic but also other magics like `ls`:

![image](https://user-images.githubusercontent.com/16781833/78611845-76c98080-782d-11ea-9c30-bfe7ab33ac29.png)

Testing with IPython the clear magic `cls` works as expected

**Comment by dalthviz** [comment]
Note: Probably we need to detect that a `cd` is being executed with an UNC path and change the actually command being executed to one using `pushd` (cmd doesn't support UNC paths by default). Using `pushd` mounts the UNC path to a drive.

See https://github.com/ipython/ipython/blob/a0c21526a456347f5f1f397dfb58bd0e7412e235/IPython/core/interactiveshell.py#L2468

**Comment by ccordoba12** [comment]
This is fixed in ipykernel **5.2.1**. Conda packages should be available soon.
